### PR TITLE
Feature/fpsmonitor

### PR DESCRIPTION
**Pull request by hardikdava**
# Description

This PR is for #255.

Testing colab: https://colab.research.google.com/drive/10dddDCD2HzzIBhzm0Zyj9NJsW0rQD5CX?usp=sharing

## Type of change

Please delete options that are not relevant.

-   [x] New feature (non-breaking change which adds functionality)
-   [x] This change requires a documentation update

## Docs

-   [x] Docs updated? What were the changes: docs/supervision/utils/fps.md and mkdocs.yaml

### CODE REVIEW

**Comment by hardikdava** [comment]
@SkalskiP This feature is ready to review.

File: supervision/utils/fps.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            >>> fps_monitor = sv.FpsMonitor()
+            >>> while True:
+            ...      (...)
+            ...      fps_monitor.tick()
+            ...      fps = fps_monitor()
+            ```
+        """
+        self.all_timestamps = deque(maxlen=sample_size)
+
+    def __call__(self, last_only=False) -> float:
```
Comment (artyaltanzaya) [inline supervision/utils/fps.py:24]: Hi there, Great job on the FpsMonitor class! This addition looks good especially for latency benchmarking.

I see that the `last_only` parameter in the` __call__` method is not being used. Is this a placeholder for future functionality, or can it be removed? Thanks for your work on this feature! 🚀

Comment (hardikdava) [reply supervision/utils/fps.py]: Hi @artyaltanzaya, thanks for pointing it out. I removed it.

**Comment by SkalskiP** [comment]
Hi @hardikdava 👋🏻! So that I understand. You accidentally committed code from two users. Accepted CLA from your main account and decided to [reopen this PR](https://github.com/roboflow/supervision/pull/280) using only your main user (the one that accepted CLA).

**Comment by hardikdava** [comment]
@SkalskiP Yeah, that is the case. but I made an updated PR from my main account. But anyhow do you know the solution for it?

**Comment by SkalskiP** [comment]
It was a good move. It is better to look for easy and fast solutions like this. And push forward. @hardikdava can that be caused by different git info configurations on different machines?

**Comment by hardikdava** [comment]
Yeah, I am suspecting the same. But it's okay. It is easier to just copy paste and pull a new pr.

**Pull request closed without merging**